networking/network_security/configuring-ipsec-ovn.adoc

```diff
@@ -12,7 +12,6 @@ IPsec is disabled by default. You can enable IPsec either during or after instal
 
 The following support limitations exist for IPsec on a {product-title} cluster:
 
-* You must disable IPsec before updating to {product-title} 4.15. There is a known issue that can cause interruptions in pod-to-pod communication if you update without disabling IPsec. (link:https://issues.redhat.com/browse/OCPBUGS-43323[*OCPBUGS-43323*])
```

Comment:
does this change also applied for other versions as well (>=4.15) ?

Reply:
Hi @pperiyasamy . This PR targets 4.18 and can be backported to 4.16, 4.17, and 4.15. I'll need to raise a separate PR for 4.19 as well.

Reply:
ok, got it, can you update the PR for the comment https://github.com/openshift/openshift-docs/pull/90760#pullrequestreview-2731708579 ? we need doc changes to be ready for 4.15.49 release.